initialization/database.js

```diff
@@ -7,8 +7,8 @@ const logger = require('~/logger/logger')
 
 const DB_OPTIONS = { useNewUrlParser: true, useUnifiedTopology: true }
 
-const databaseInitialization = () => {
-  mongoose.connect(MONGODB_URL, DB_OPTIONS)
+const databaseInitialization = async () => {
+  await mongoose.connect(MONGODB_URL, DB_OPTIONS)
```

Comment:
why didn't remove options as you wanted?

Reply:
I didn't really want to remove them, I just wanted to add the promise usage regardless

Reply:
fixed